### PR TITLE
code to test JS best practices

### PR DESCRIPTION
**Pull request by Besufekad-HAZ**
-Test DRY, KISS, and YAGNI.
-Test JS best practices.
-Test HTML and CSS best practices.

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -68,7 +68,7 @@ form {
 input,
 select,
 textarea {
-  color: #000;
+  color: rgb(0, 0, 0);
 }
```
Comment (Kevin-Mena) [inline src/style.css:73]: -This first item must be of index 1 and then you scale... So five should be index 5 not 4 https://github.com/Besufekad-HAZ/To-do-list-testing/blob/602d0e17fce5ee84fd9db5202e6cbeb7cf267c9b/src/tasks-list.js#L17-L25

**Review by Kevin-Mena — COMMENTED** [review]
@Besufekad-HAZ  Your project is awesome and you did a great job implementing the project requirements.
**Highlights**
✅ Javascript best practices followed
✅ DRY, KISS and YAGNI rules followed
✅ HTML and CSS best practices

**OPTIONAL SUGGESTIONS**
Check the comments below👇

**Review by Kevin-Mena — APPROVED** [review]
### STATUS: APPROVED 🎉 👏 🍾
@Besufekad-HAZ Your project is awesome and you did a great job implementing the project 😃 👏 ! There is nothing else to say other than... it's time to merge it:
Congratulations! 🎉.
 
**Highlights**
✅ Javascript best practices followed
✅ DRY, KISS, and YAGNI rules followed
✅ HTML and CSS best practices

**Comment by Besufekad-HAZ** [comment]
Thanks for the review. 👍 👍 👍

**Pull request merged**